frontend/src/utils/format_column_data.ts

```
@@ -20,255 +20,152 @@ export type Edge = {
+      que[0].push(top_level[i]);
+      rows[0].push(top_level[i]);
+    }
 
-    FIXME: Is not true anymore due to fe fiddling
+    // Loop through and get all decendents in a breadth first search
+    while (curr_q_idx < que.length) {
+      const next_column: string[] = [];
+      const current_column = que[curr_q_idx];
+
+      for (let i = 0; i < current_column.length; i++) {
+        const has_children = this.adj_list.get(current_column[i])!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)